cts/schemas/test-3/ref/ping-node.ref-0

```diff
@@ -0,0 +1,24 @@
+<cib crm_feature_set="3.19.7" validate-with="pacemaker-3.10" epoch="2" num_updates="0" admin_epoch="0" original="1">
+  <configuration original="1">
+    <!-- The essential elements of this test are:
+         * There are three nodes:
+           * node1: member
+           * node2: ping
+           * node3: remote
```

Comment:
What's the difference between the various tests?

Reply:
I'm not sure I understand the question. This commit just creates the tests. Nothing happens until the next commit.

Reply:
There are multiple tests that at a cursory glance look identical

Reply:
I'm only seeing the `ping-node` test and its sequence of outputs. What are the multiple tests?

If you mean multiple conditions within this one test, "member" and "remote" are only there to ensure that nothing changes for those nodes.

Reply:
So are cts/schemas/test-3/ref/ping-node.ref-0, ping-node.ref-1, ping-node.ref-2, etc. just the result after each stage of the transformation?

Reply:
Yes